pkg/cli/delete.go

```diff
@@ -0,0 +1,59 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/rest"
+
+	rancher "github.com/torchiaf/kubectl-rancherx/pkg/rancher"
+)
+
+func delete(client *Client) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:           "delete",
+		Short:         "Delete Rancher resources by resources and names.",
+		Args:          cobra.ExactArgs(1),
+		RunE:          ValidateSubCommand(resources),
```

Comment:
I don't think this is needed. You can provide no `Run/RunE` and it will "fail". Maybe only if you want to have some custom error.

Reply:
Yes, the wanted behavior was to get errors in case no commands are defined yet for sub-resouces (ATM we defined only `projects`).